tests/scripts/test_translate_ciphers_format.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/sh
+
+# test_translate_ciphers_format.sh
+#
+# Copyright The Mbed TLS Contributors
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Purpose
+#
+# Test translate_ciphers.py formatting by comparing the translated
```

Comment:
It was designed as more of a preliminary check to make sure it can translate multiple ciphers at once, before I implemented it in `compat.sh`. `test_translae_ciphers_names.py` does them one by one, it was a precursor as sorts before I had assembled everything. I was going to suggest that once the PR was ready to go, we could probably scrap this test, but keep the other one. It essentially just mimic's compat.sh

Reply:
After translating the code to python for unit tests it became redundant so it has been removed now